packages/astro/src/snippets.ts

```diff
@@ -23,6 +28,7 @@ Spotlight.init({
   ],
   showTriggerButton: ${options.showTriggerButton === false ? 'false' : 'true'},
   injectImmediately: ${options.injectImmediately === true ? 'true' : 'false'},
+  fullPage: ${options.fullPage === true ? 'true' : 'false'},
```

Comment:
Curious, why don't we do `Boolean(options.fullPage).toString()`? (and for all the others?)

Or even better, something like:

```js

const init_options = {
  integrations: integrationCalls,
  ...options,
};

return `Spotlight.init(${JSON.stringify(init_options))})`
```